daemons/controld/controld_execd.c

```diff
@@ -523,6 +525,13 @@ build_parameter_list(const lrmd_event_data_t *op,
         struct ra_param_s *param = (struct ra_param_s *) iter->data;
         bool accept = pcmk_is_set(param->rap_flags, param_type);
 
+        if (param_type == ra_param_reloadable) {
+            /* For the restart digest, we want to select parameters that are not
+             * reloadable, for both the string and XML.
+             */
+            accept = !accept;
+        }
+
```

Comment:
Yep - probably the most efficient code.
But having one switch in the beginning that sorts out the logic for an accept_xml and an accept_list might be easier to understand. I know it is orthogonal cases that negate accept but that may change in the future, code might be reordered ... just leaves a feeling of danger ;-)

Reply:
Makes sense

Reply:
Added commit on top

Reply:
guess this is way quicker to grasp for the first-time-reader - although leaving the comments in might have made it still a bit easier ;-)

Reply:
The explanation is in the doxygen comment, and also at the caller -- I hoped that was enough :)